docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -147,6 +147,13 @@ This prop is available on all fields and pickers.
 
 {{"demo": "CustomFieldFormat.js"}}
 
+### Dense field format
```

Comment:
Although when you read the full page, it does make sense to have this section here, if `dense` is not the default, I'm not sure that `react-date-pickers/adapters-locale/` would be the place that users would intuitively look for this option. 
Would it make sense to (also) have it on visual customizations grouped with other props that have minor visual impacts on the component?

Reply:
If you are in favor of using `dense` as the default, I'm fine with it.

But if we take a moment to think about the localization of the doc.
I agree that it's kind of related to both the format and the visual aspect, @LukasTy @alexfauquette what's your opinion on this one ?

Reply:
I do also think that if a user might be looking for such customization—visual customization might be the first place they'd go. 🤔 
What do you think about putting it there, but also adding a section/callout in `adapters-locale` linking there? 🤔

Reply:
I'll do that :+1: